src/komponenter/NyttIAppen/nyheter.tsx

```diff
@@ -1,10 +1,24 @@
 import EksternLenke from '@/komponenter/navigation/EksternLenke';
-import React from 'react';
 
 const tilDato = (dag: number, måned: number, år: number, timer: number = 0, minutter: number = 0) =>
     new Date(år, måned - 1, dag, timer, minutter);
 
 const nyheter = [
+    {
+        dato: tilDato(15, 1, 2024),
+        tittel: 'Varsling av deltakere endret',
+        innhold: (
+            <>
+                <p>
+                    Deltaker vil nå bli automatisk varslet på min side på nav.no i form av oppgaver og beskjeder i tilegg til sms ved følgende hendelser:
+                    <li>Når arbeidsgiver godkjenner (oppgave om å godkjenne)</li>
+                    <li>Når avtalen er inngått (beskjed)</li>
+                    <li>Når avtalen blir forlenget (beskjed)</li>
+                    <li>Når avtalen blir forkortet (beskjed)</li>
+                </p>
+            </>
```

Comment:
Du kan vel kanskje droppe Fragment her, siden det bare er 1 child (`<p>`)?

`<li>` må og være [wrappet i en `ul`, `ol` eller `menu`](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-li-element.html).

Reply:
mange gode poenger